File: solutions/python/rna-transcription/1/rna_transcription.py
"""A simple function for transcribing DNA to RNA"""

def to_rna(dna_strand):
    rna_strand = ""
    for nucleotide in dna_strand:
        match nucleotide:
            case "G":
                rna_strand += "C"
            case "C":
                rna_strand += "G"
            case "T":
                rna_strand += "A"
            case "A":
                rna_strand += "U"
    return rna_strand

File: solutions/kotlin/rna-transcription/1/src/main/kotlin/RnaTranscription.kt
fun transcribeToRna(dna: String): String {
    var rna = ""
    for (nucleotide in dna) {
        when (nucleotide) {
            'G' -> rna += "C"
            'C' -> rna += "G"
            'T' -> rna += "A"
            'A' -> rna += "U"
        }
    }
    return rna
}
